### PR TITLE
Bombchu OK

### PR DESCRIPTION
**Pull request by fig02**
floats suck
no docs cause math can fuck off (ty whoever ends up doing it)
FeelsOkayMan

edit: petrie matched it

### CODE REVIEW

File: src/overlays/actors/ovl_En_Bom_Chu/z_en_bom_chu.c [hunk 1]
```diff
@@ -48,29 +52,440 @@ static ColliderJntSphInit D_809C6D54 = {
...
+        }
+    }
+
+    this->actionFunc = EnBomChu_WaitForKill;
+}
+
+void func_809C5B38(Vec3f* arg0, Vec3f* arg1, Vec3f* dest) {
+    dest->x = (arg0->y * arg1->z) - (arg0->z * arg1->y);
+    dest->y = (arg0->z * arg1->x) - (arg0->x * arg1->z);
+    dest->z = (arg0->x * arg1->y) - (arg0->y * arg1->x);
```
Comment (krimtonz) [inline src/overlays/actors/ovl_En_Bom_Chu/z_en_bom_chu.c:130]:
```suggestion
void EnBomChu_Cross(Vec3f* a, Vec3f* b, Vec3f* dest) {
    dest->x = (a->y * b->z) - (a->z * b->y);
    dest->y = (a->z * b->x) - (a->x * b->z);
    dest->z = (a->x * b->y) - (a->y * b->x);
```

File: src/overlays/actors/ovl_En_Bom_Chu/z_en_bom_chu.c [hunk 2]
```diff
@@ -48,29 +52,440 @@ static ColliderJntSphInit D_809C6D54 = {
...
+    if (this->timer != 0) {
+        this->timer--;
+    }
+
+    if (this->timer == 0) {
+        EnBomChu_Explode(this, globalCtx);
+        return;
+    }
+
+    if (Actor_HasNoParent(&this->actor, globalCtx)) {
```
Comment (krimtonz) [inline src/overlays/actors/ovl_En_Bom_Chu/z_en_bom_chu.c:205]: Petition to rename this function the batman function?

**Review by krimtonz — APPROVED** [review]
Just a naming suggestion.

File: src/overlays/actors/ovl_En_Bom_Chu/z_en_bom_chu.c (hunk 1 above)
Comment (fig02) [reply src/overlays/actors/ovl_En_Bom_Chu/z_en_bom_chu.c:130]: thanks, if you have any other suggestions for the math stuff lmk. either func names or var names

**Review by Roman971 — APPROVED** [review]

**Pull request merged**